src/assets/scripts/client/parsers/aircraftCommandParser/aircraftCommandMap.js

```diff
@@ -54,67 +20,191 @@ const UNICODE_COMMANDS = {
-    d: 'altitude',
-    altitude: 'altitude',
-    a: 'altitude',
-    takeoff: 'takeoff',
-    to: 'takeoff',
-    cto: 'takeoff',
-    fph: 'flyPresentHeading',
-    heading: 'heading',
-    fh: 'heading',
-    h: 'heading',
-    turn: 'heading',
-    t: 'heading',
-    speed: 'speed',
-    slow: 'speed',
-    sp: 'speed',
-    '+': 'speed',
-    '-': 'speed',
-    ils: 'land',
-    i: 'land',
-    land: 'land',
-    l: 'land',
-    '*': 'land',
-    reroute: 'reroute',
-    rr: 'reroute',
-    route: 'route',
-    f: 'fix',
-    fix: 'fix',
-    track: 'fix',
-    direct: 'direct',
-    pd: 'direct',
-    dct: 'direct',
-    hold: 'hold',
-    squawk: 'squawk',
-    sq: 'squawk',
-    delete: 'delete',
-    del: 'delete',
-    kill: 'delete',
-    sa: 'sayAltitude',
-    saa: 'sayAssignedAltitude',
-    sh: 'sayHeading',
-    sah: 'sayAssignedHeading',
-    si: 'sayIndicatedAirspeed',
-    sas: 'sayAssignedSpeed'
+    abort: {
+        aliases: ['abort'],
+        functionName: 'runAbort',
+        isSystemCommand: false
+    },
+    airport: {
+        aliases: ['airport'],
+        functionName: '',
+        isSystemCommand: true
+    },
+    altitude: {
+        aliases: ['a', 'altitude', 'c', 'climb', 'd', 'descend'],
+        functionName: 'runAltitude',
+        isSystemCommand: false
+    },
+    auto: {
+        aliases: ['auto'],
+        functionName: '',
+        isSystemCommand: true
+    },
+    clear: {
+        aliases: ['clear'],
+        functionName: '',
+        isSystemCommand: true
+    },
+    clearedAsFiled: {
+        aliases: ['caf', 'clearedAsFiled'],
+        functionName: 'runClearedAsFiled',
+        isSystemCommand: false
+    },
+    climbViaSid: {
+        aliases: ['climbViaSid', 'cvs'],
+        functionName: 'runClimbViaSID',
+        isSystemCommand: false
+    },
+    delete: {
+        aliases: ['del', 'delete', 'kill'],
+        functionName: 'runDelete',
+        isSystemCommand: false
+    },
+    descendViaStar: {
+        aliases: ['descendViaStar', 'dvs'],
+        functionName: 'runDescendViaStar',
+        isSystemCommand: false
+    },
+    direct: {
+        aliases: ['dct', 'direct', 'pd'],
+        functionName: 'runDirect',
+        isSystemCommand: false
+    },
+    expectArrivalRunway: {
+        aliases: ['e'],
+        functionName: 'runExpectArrivalRunway',
+        isSystemCommand: false
+    },
+    fix: {
+        aliases: ['f', 'fix', 'track'],
+        functionName: 'runFix',
+        isSystemCommand: false
+    },
+    flyPresentHeading: {
+        aliases: ['fph'],
+        functionName: 'runFlyPresentHeading',
+        isSystemCommand: false
+    },
+    heading: {
+        aliases: ['fh', 'h', 'heading', 't', 'turn'],
+        functionName: 'runHeading',
+        isSystemCommand: false
+    },
+    hold: {
+        aliases: ['hold'],
+        functionName: 'runHold',
+        isSystemCommand: false
+    },
+    land: {
+        aliases: ['\\u2B50', '*', 'i', 'ils'],
```

Comment:
are we still handling unicode commands?

Reply:
No; I believe if an arrow key is pressed it's replaced with `c` or `d` in the command bar. So a Unicode character would never be sent to be parsed.

Reply:
this is the star that's inserted when pressing the numpad `*`. I'm not sure if it's in use, was one of the loose ends to be cleaned up when I get to it. Still a work in progress of course.

Reply:
it was part of a `UNICODE_COMMANDS` enum that was `...`ed into the `AIRCRAFT_COMMANDS_MAP`. so i haven't changed anything really, but nonetheless intend to find out if it should be removed or not.

Reply:
not really needed. removed, and redirected the hotkey to insert the `*` alias.